games/models.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+
+from django.core.validators import MaxValueValidator
+from django.db import models
+from django.utils.translation import ugettext as _
+from model_utils.models import TimeStampedModel
+
+from foosball.users.models import User
+
+
+class Game(TimeStampedModel):
+    played_on = models.DateTimeField(
+        default=datetime.datetime.now,
+        verbose_name=_("played on"),
+        help_text=_("Time when the game was played")
+    )
+
+    class Meta:
+        ordering = ['-played_on']
+
+    def __str__(self):
+        return " - ".join(map(str, self.teams.all()))
+
+
+class GameTeam(models.Model):
+    game = models.ForeignKey(Game, related_name="teams", on_delete=models.CASCADE)
+    score = models.PositiveSmallIntegerField(
+        verbose_name=_("score"),
+        help_text=_("Team's score in the game"),
+        validators=[MaxValueValidator(limit_value=10)]
+    )
+    players = models.ManyToManyField(User)
```

Comment:
This structure for games and game teams fails to answer the most important question of all: which side is better, blue or red. How about adding two (one to one) foreign keys from game to game team?